File: JS/UberBlack.mjs
import Car from './Car.mjs'

class UberBlack extends Car
{
    constructor(license, driver, typeCarAccepted, seatsMaterial)
    {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}

export default UberBlack;

File: Python/UberBlack.py
import this
from car import Car

class UberBlack(Car):
    typeCarAccepted= []
    seatsMaterial=[]

    def __init__(self, license, driver, typeCarAccepted, seatsMaterial):
        super().__init__(license, driver)
        this.typeCarAccepted= typeCarAccepted
        this.seatsMaterial= seatsMaterial
